src/app/test/test.component.ts

```diff
@@ -48,6 +48,7 @@ export class TestComponent implements OnInit, AfterViewInit {
     storageName: 'Test',
     targetStorage: '',
   };
+  actionControl: FormControl<string> = new FormControl();
```

Comment:
Any reason you don't just have a variable for the value and a method to change that value? Why is it a form control in this case?

Reply:
I didn't know exactly what you meant in the issue by "Angular's built in form support" and this was what I came accross when doing some research on Angular forms. I see now that it's not that useful for this scenario.